ci-templates/gitlab/exporter.yml

```diff
@@ -15,7 +23,7 @@ export-to-t4c:
     FILE_HANDLER: local
 
     # Semantic Capella version (x.x.x) >= 6.0.0
-    # Defaults to ${CAPELLA_VERSION}-${CAPELLA_DOCKER_IMAGES_REVISION} if not defined
+    # Defaults to 6.0.0 if not defined
     CAPELLA_VERSION: 6.0.0
```

Comment:
In the `diagram-cache.yml` this line is commented out as it defaults to 6.0.0. I would either also comment it out here or not comment it out there.

Reply:
Done.